tidypolars/funs.py

```diff
@@ -131,6 +131,22 @@ def as_string(x):
     x = _col_expr(x)
     return x.cast(pl.Utf8)
 
+def as_factor(x, levels):
```

Comment:
This should probably be called `as_categorical()`? I have been trying to keep the column types to follow polars instead of R naming conventions.

For example - I have `as_string()` instead of `as_character()` and `as_boolean()` instead of `as_logical()`

Reply:
Both, then? I think it makes sense, but if the goal is to use all verbs from tidyverse (mutate, etc.), why not follow the R naming convention for some of the key data wrangling functions too, as it is done for str_*? I would use an alias and both methods.

Reply:
I'm good having both. Also can we define the default as `levels = None` or something so that when the user just does `as_factor(x)` it uses `levels = x.unique()`

Something like this
```python
def as_factor(x, levels = None):
    if levels == None:
        levels = x.unique()
```

Reply:
good idea